.env.telostestnet

```diff
@@ -40,6 +40,7 @@ HAPI_SYNC_PRODUCERS_INTERVAL=86400
 HAPI_SYNC_PRODUCER_INFO_INTERVAL=1
 HAPI_SYNC_PRODUCER_CPU_INTERVAL=6
 HAPI_SYNC_EXCHANGE_RATE=86400
+HAPI_COINGECKO_API_TOKEN_ID=telos
```

Comment:
@JustinCast You are adding this new variable for telos environment file but not for the other networks please add the proper values for other networks to avoid a problem when someone else try to run the project the same for `HAPI_EOS_EXCHANGE_RATE_API `

Reply:
Thank you for the comments!